caravel/templates/listing_show.html

```diff
@@ -18,28 +18,26 @@
           {% else %}
             <div class="item">
           {% endif %}
-
-          {% if photo %}
             <img src="{{ photo | public_url }}">
-          {% else %}
-            <div class="large-image-placeholder">
-              {{ listing.body[:1].upper() }}
-            </div>
-          {% endif %}
-
-{#          Folloring div closes item active or item#}
           </div>
         {% set i = i + 1 %}
         {% endfor %}
+        {% if i == 0 %}
```

Comment:
I dont think it looks better without the no-image image. 
<img width="1177" alt="screen shot 2015-09-23 at 10 28 51 am" src="https://cloud.githubusercontent.com/assets/12107160/10052948/3d549c1e-61de-11e5-9430-f093fa3e74e1.png">